components/Dashboard/Modals/ProfileDialog/index.js

```diff
@@ -138,58 +144,99 @@ const ProfileDialog = ({ session, isOpen, handleClose, tabs }) => {
+                {/* Settings Nav for Small Screens */}
+                {showSettings ? (
+                  <Hidden mdUp>
+                    <Grid container item xs={12} md={3} className="p-4">
+                      <TabList className="w-full">
+                        {tabs.map((tab, index) => (
+                          <Tab key={tab.tabName} className="pl-2">
+                            <Button
+                              fullWidth
+                              disableRipple
+                              onClick={toggleSettings}
+                              className={classNames([
+                                'justify-start',
+                                'pb-2',
+                                'pl-4',
+                                classes.tab,
+                                index === tabIndex && classes.activeTab,
+                              ])}>
+                              {tab.tabName}
+                            </Button>
+                          </Tab>
+                        ))}
+                      </TabList>
+                      <Grid container direction="column" alignItems="baseline" justify="flex-end">
+                        <Button>Privacy</Button>
+                        <Button>Logout</Button>
+                      </Grid>
+                    </Grid>
                   </Hidden>
-                  <Grid item xs={12} className={classes.mainFormBody}>
-                    {renderTabComponent(tab.component)}
+                ) : (
+                  /* GREY AREA */
+                  <Grid container item direction="row" xs={12} className="p-4">
+                    <Hidden mdUp>
+                      <ArrowBackIosIcon onClick={toggleSettings}></ArrowBackIosIcon>
+                    </Hidden>
+                    <Typography variant="h3" gutterBottom className={classes.tabTitle}>
+                      {tab.title}
+                    </Typography>
+                    <Hidden smDown>
+                      <CloseIcon className={classes.closeIcon} />
+                    </Hidden>
+                    <Grid item xs={12} className={classes.mainFormBody}>
+                      {renderTabComponent(tab.component)}
+                    </Grid>
                   </Grid>
-                </Grid>
+                )}
               </TabPanel>
             ))}
-            <Grid container item justify="flex-end" className="p-4">
-              <Button
-                variant="contained"
-                color="primary"
-                className={classes.containedPrimary}
-                startIcon={editMode ? <CheckIcon /> : <EditIcon />}
-                onClick={toggleSaveMode}
-                disabled={isEditButtonDisabled()}>
-                {editMode ? 'Save' : 'Edit Profile'}
-              </Button>
-            </Grid>
+            {showSettings ? null : (
+              <Grid container item justify="flex-end" className="p-4">
```

Comment:
conditional renders are usually written with `&&`. So in this case u can do `!showSettings && <Grid...>`
Since components always evaluate to truthy, this part will only show when `showSettings === false`